filebeat/harvester/reader/timeout.go

```diff
@@ -64,6 +64,6 @@ func (p *Timeout) Next() (Message, error) {
 		}
 		return msg.line, msg.err
 	case <-time.After(p.timeout):
-		return Message{}, p.signal
+		return NewMessage(), p.signal
```

Comment:
My idea here that it is up to the caller what he does with the message, so the message should be properly initialised.